rapidpro/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
+
+    <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%d %t %-5p [%c] %m%n"/>
+        </layout>
+    </appender>
+
+    <logger name="archetype.test.service">
+        <level value="WARN"/>
+    </logger>
+    <logger name="org.springframework.web">
+        <level value="ERROR"/>
+    </logger>
+
+    <root>
+        <priority value="ERROR"/>
+        <appender-ref ref="CONSOLE"/>
+    </root>
+
+</log4j:configuration>
```

Comment:
Yes, the next PR will have unit tests for this current PR and for everything introduced in the next one.